docs/api-reference/queries.md

```diff
@@ -133,12 +133,21 @@ SELECT * FROM table WHERE name = '{{defaultOrValue "field1" "gopher"}}';
 
 ### inFormat
 
-If value of param is an slice this function format to an IN SQL clause.
+If you need to format data for a usage on a `IN ('option1', 'option2')` statement.
```

Comment:
How does these changes on this file relates to this PR, can you clarify?

Reply:
It's not related to this PR. But it's needed for a better comprehension of the function, I've done this while solving a work issue related to this function and forgot to remove the commit from the stash.

Reply:
Ok, thanks for the clarification. I just found it confusing given the context : )

Reply:
Just checking: you marked that as _resolved_, but you keep unrelated PR changes. Is that correct? If so, what is the reasoning?

Reply:
The only scenario in which this would make sense is the one in which these changes, even if unrelated, are needed.

If that's the case, you should open another branch and PR for these changes. Since we raised that 2 weeks ago, if these changes are needed, and if they were in a dedicated PR, they would probably be merged already by now.